install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -127,10 +127,11 @@ $ oc set volume deploymentconfigs/docker-registry --add --name=registry-storage
      --claim-name=<pvc_name> --overwrite
 ----
 
-[NOTE]
+[IMPORTANT]
 ====
-See xref:registry_known_issues.adoc#install-config-registry-known-issues[Known Issues] if using a scaled registry with a
-shared NFS volume.
+Testing has shown issues with using the NFS server on RHEL as storage backend for the container registry. This includes the embedded registry and Quay, Cassandra for metrics storage, and ElasticSearch for logging storage. Therefore, using NFS on RHEL as storage for these framework services is not recommended.
```

Comment:
We can remove the references to Cassandra, and Elastic Search here. 

The section this is in, implies we will not be discussing those components.

Reply:
I removed these in the new PR.